tests/ci/cdk/cdk/aws_lc_analytics_stack.py

```diff
@@ -28,6 +28,9 @@ def __init__(self,
             webhook=True,
             webhook_filters=[
                 codebuild.FilterGroup.in_event_of(codebuild.EventAction.PUSH)
+                # The current FIPS branch does not have the configuration needed to run the analytics, once we update
+                # the branch or create a new FIPS branch it should be updated to '(main)|(fips.*)'
+                .and_branch_is("main")
```

Comment:
NIT: maybe `fips-*` can be added earlier before cutting the FIPS branch to avoid the deployment.